app/players/dwarves/dwarf_blacksmith.py

```diff
@@ -0,0 +1,18 @@
+from app.players.dwarves.dwarf import Dwarf
+
+
+class DwarfBlacksmith(Dwarf):
+
+    def __init__(self, nickname: str,
+                 favourite_dish: str,
+                 skill_level: int
+                 ) -> None:

```

Comment:
You haven’t fixed indentation
```
    def __init__(
             self,
             nickname: str,
             favourite_dish: str,
             skill_level: int
     ) -> None:
```

Reply:
There is an error in your proposed solution Flake8 "Closing bracket does not match visual indentation"